From 30bc827610610c925015f2a97a0280b3cf347b07 Mon Sep 17 00:00:00 2001
From: Rakesh Iyer <riyer@nvidia.com>
Date: Wed, 25 Jul 2012 14:14:03 -0700
Subject: [PATCH] Enable use of ro.sf.lcd_density prop by system user

This allow gralloc to initialize the lcd_density value under certain conditions.

bug 1014380

Change-Id: I014825eb5342a834af26863f9fbe88afe24559ab
Signed-off-by: Rakesh Iyer <riyer@nvidia.com>
Reviewed-on: http://git-master/r/118434
Reviewed-on: http://git-master/r/142765
GVS: Gerrit_Virtual_Submit
Reviewed-by: Prajakta Gudadhe <pgudadhe@nvidia.com>
---
 init/property_service.c |    1 +
 1 file changed, 1 insertion(+)

diff --git a/init/property_service.c b/init/property_service.c
index ef9f9bd..f46accd 100755
--- a/init/property_service.c
+++ b/init/property_service.c
@@ -95,6 +95,7 @@ struct {
     { "persist.service.bdroid.", AID_BLUETOOTH,   0 },
     { "selinux."         , AID_SYSTEM,   0 },
     { "modem.",            AID_RADIO,   0},
+    { "sf.lcd_density",   AID_SYSTEM,   0 },
     { NULL, 0, 0 }
 };
 
-- 
1.7.9.5

